src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportControlLoader.java

```diff
@@ -227,6 +251,26 @@ private ImportControl getRoot() {
     }
 
     /**
+     * Utility to get a strategyOnMismatch property.
+     * @param attributes collect to get attribute from.
+     * @return the value of the attribute.
+     */
+    private static MismatchStrategy getStrategyOnMismatch(final Attributes attributes) {
+        final String returnValue = attributes.getValue(STRATEGY_ON_MISMATCH_ATTRIBUTE_NAME);
+        MismatchStrategy strategyOnMismatch = null;
+        if ("delegateToParent".equals(returnValue)) {
+            strategyOnMismatch = MismatchStrategy.DELEGATE_TO_PARENT;
+        }
+        else if ("allowed".equals(returnValue)) {
+            strategyOnMismatch = MismatchStrategy.ALLOWED;
+        }
+        else if ("disallowed".equals(returnValue)) {
+            strategyOnMismatch = MismatchStrategy.DISALLOWED;
+        }
+        return strategyOnMismatch;
```

Comment:
I do not like that you return NULL there, NULL is old way how engineer described uncertainty.
we always know strategy, there is default. NULL is not allowed